TLSharp.Core/TelegramClient.cs

```diff
@@ -110,15 +112,15 @@ public bool IsUserAuthorized()
 
             var authCheckPhoneRequest = new TLRequestCheckPhone() { phone_number = phoneNumber };
             var completed = false;
-            while(!completed)
+            while (!completed)
```

Comment:
Even if not your fault, if you want to make whitespace changes you have to propose them in a different commit or PR ;)